src/components/dashboard/Settings.js

```diff
@@ -61,10 +65,88 @@ const PrivacyOption = ({ title, value, field, setPrivacy }) => {
     </RadioButton.Group>
   )
 }
+const supportedCountryCodes = ['US', 'GB', 'ES', 'FR', 'IT', 'KR', 'BR', 'UA', 'TR', 'VN', 'CN', 'IN', 'ID', 'AR']
```

Comment:
@L03TJ3 @johnsmith-gooddollar isn't there a way to get the available languages dynamically and not hardcoded?

Reply:
well above is basically a left-over from a previous attempted lib/component and it needed to be mapped correctly to the countries.

We could do it automatically (we still need to define labels somewhere.)

do any of you know where the list in the build file comes from: (src/languages/locales/index.js -- its in .gitignore so I cannot link here)